source/qml/Samples/QmlVisualizer/adaptivecard_dark_config.h

```diff
@@ -27,7 +27,7 @@ namespace DarkConfig
             }
         },
         "monospace":{
-            "fontFamily":"'Courier New', Courier, monospace",
+            "fontFamily":"Courier New, Courier, monospace",
```

Comment:
With the single quotes, monospace font was working for Qml Text element, but not for the rich text (html tag) in richTextBlock
On removal of single quotes, monospace is working for both TextBlock and RichTextBlock